scripts/proto_gen.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
I may introduce a cmake way to generate pb.cc from .proto file, after this is merged.

Reply:
Ok